ios/AirshipFrameworkProxy/Proxies/AirshipChannelProxy.swift

```diff
@@ -28,6 +28,24 @@ public class AirshipChannelProxy {
             editor.remove(tags)
         }
     }
+    
+    @objc
+    public func editTags(json: Any) throws {
+        let data = try JSONUtils.data(json)
+        let operations = try JSONDecoder().decode(
+            [TagOperation].self,
+            from: data
+        )
```

Comment:
same here, what happens if we try to feed it with an invalid json?

Reply:
We bubble it up to the framework that uses it. Usually its a promise type API that we can provide the error